CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Lotus changelog
 
+# 1.10.1 / 2021-07-05
+
+This is an optional but **highly recommended** release of Lotus that have many bug fixes and improvements based on the feedbacks we got from the community since HyperDrive.
```

Comment:
```suggestion
This is an optional but **highly recommended** release of Lotus that has many bug fixes and improvements based on the feedback we got from the community since HyperDrive.
```

Reply:
Is it fair to say this release is only recommend for miners?

Reply:
Yes it is